jekyll/_cci2/insights-glossary.md

```diff
@@ -69,7 +69,11 @@ additional context to Insights metrics by presenting a relative benchmark
 against previous performance. For instance, on the last 7-day view, trends will
 display the change in value or delta compared to the prior 7-day window.
 
-Trends are calculated as `100 * (current value - previous value) / prior-value`.
+Trends displayed in the CircleCI UI are calculated as `100 * (current value - previous value) / prior-value`.
+
+Trends received from the [CircleCI API](https://circleci.com/docs/api/v2/index.html#operation/getProjectWorkflowsPageData) are calculated as a ratio instead of a percentage with the following formula: `(current-value / prior-value)`.  These trends are 1-based and not 0-based.
```

Comment:
```suggestion
Trends received from the [CircleCI API](https://circleci.com/docs/api/v2/index.html#operation/getProjectWorkflowsPageData) are calculated as a ratio instead of a percentage with the following formula: `(current-value / prior-value)`. These trends are 1-based and not 0-based.
```